sdk/ml/azure-ai-ml/azure/ai/ml/entities/_registry/registry.py

```diff
@@ -189,6 +189,11 @@ def _convert_yaml_dict_to_entity_input(
             if global_acr_exists:
                 if not hasattr(region_detail, "acr_details") or len(region_detail.acr_details) == 0:
                     region_detail.acr_config = [acr_input]
+        if INTELLECTUAL_PROPERTY in input:
+            intellectual_property = input.pop(INTELLECTUAL_PROPERTY)
+            publisher = intellectual_property.get("publisher", None)
+            if publisher:
+                input["intellectual_property_publisher"] = publisher
```

Comment:
have we verified that we want this difference between SDK/CLI contracts? Once we GA both, we cannot change them.

Reply:
synced offline. Given registry GA has been delayed, we can address this later.